README.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
Sorry for the late reply, I've been working on it but forgot to reply.
I am currently rewriting the report so that it is more interactive and has less heading levels.

So far I've finished rewriting the methods section and will work on the results  section, which I suspect will be much longer when I try to explain the functions.

Thank you